mods/cnc/chrome/ingame-chat.yaml

```diff
@@ -19,7 +19,7 @@ Container@CHAT_PANEL:
 				TextNotificationsDisplay@CHAT_DISPLAY:
 					Width: PARENT_RIGHT
 					Height: PARENT_BOTTOM
-					RemoveTime: 250
+					Duration: 10000
```

Comment:
This will need an update rule.

Reply:
Oh, I didn't know we can update chrome yaml. With `UpdateChromeNode`? Seems like it has never been used :crossed_fingers: